test/commons/dom/get-target-rects.js

```diff
@@ -74,10 +76,28 @@ describe('get-target-rects', () => {
       </button>
     `);
     const rects = getTargetRects(vNode);
-    console.log(JSON.stringify(rects));
     assert.deepEqual(rects, [
       new DOMRect(10, 5, 30, 7),
       new DOMRect(10, 5, 7, 40)
     ]);
   });
+
+  (shadowSupport ? it : xit)(
```

Comment:
We stopped doing this now that we're not testing against IE.

```suggestion
  it(
```